feat: adding translation image, static, and bypass templates (#543)
aziontech/docs: src/content/docs/pt-br/pages/guias/marketplace/templates/image-suite/image-suite.mdx

---
title: Como implantar o template Suite Image Cache

> description: This template includes configurations for optimizing the delivery and performance of static content.

Author: ```suggestion
description: O template inclui configurações para aprimorar o carregamento de imagens e o armazenamento em cache com base na extensão do arquivo..
```

meta_tags: templates, guides, Azion Marketplace
namespace: docs_guides_image_cache_template
permalink: /documentacao/produtos/guias/suite-image-cache/
---

import Badge from '~/components/Badge.astro';

<Badge variant="accent">
Preview
</Badge>

> **Suite Image Cache** inclui configurações para aprimorar o carregamento de imagens e o armazenamento em cache com base na extensão do arquivo. Essas configurações utilizam regras avançadas de chave de cache aplicadas por meio do Rules Engine para Edge Application.

Author: ```suggestion
**Suite Image Cache** inclui configurações para aprimorar o carregamento de imagens e o armazenamento em cache com base na extensão do arquivo. Essas configurações utilizam regras avançadas de cache key aplicadas por meio do Rules Engine para Edge Application.
```

---
title: Como implantar o template Suite Image Cache
description: This template includes configurations for optimizing the delivery and performance of static content.
meta_tags: templates, guides, Azion Marketplace
namespace: docs_guides_image_cache_template
permalink: /documentacao/produtos/guias/suite-image-cache/
---

import Badge from '~/components/Badge.astro';

<Badge variant="accent">
Preview
</Badge>

**Suite Image Cache** inclui configurações para aprimorar o carregamento de imagens e o armazenamento em cache com base na extensão do arquivo. Essas configurações utilizam regras avançadas de chave de cache aplicadas por meio do Rules Engine para Edge Application.

> Você deve vincular sua origem ao template para começar a usar este conjunto de regras de cache para entrega. Após o deployment, o script cria uma nova aplicação edge que é executada na Plataforma de Edge Computing da Azion, com a qual você pode gerenciar e ajustar as configurações.

Reviewer: ```suggestion
Você deve vincular sua origem ao template para começar a usar este conjunto de regras de cache para entrega. Após o deployment, o script cria uma nova edge application que é executada na Plataforma de Edge Computing da Azion, com a qual você pode gerenciar e ajustar as configurações.
```

Campos identificados com um asterisco são obrigatórios.

- **Application Name** *: o nome da sua edge application na Azion.
    - Digite um nome único e fácil de lembrar. Se o nome já tiver sido usado, a plataforma retornará uma mensagem de erro.
- **Origin Address** *: o endereço da origem de onde a edge application deve buscar o conteúdo original.
    - Exemplo: `www.yourwebsite.com`
- **Custom Host Header**: em alguns casos, pode ser necessário adicionar informações do host no cabeçalho da solicitação para recuperar informações da origem. Esta variável é usada para esse propósito.
    - Exemplo: `${host}`. Saiba mais sobre o host header e como customizá-lo no [documentação de Origins](/pt-br/documentacao/produtos/edge-application/origins/#host-header).
- **Static files extensions to cache** *: as extensões de arquivo para as regras, separadas por uma vírgula. Você pode adicionar quantas variáveis precisar, com base no exemplo.
    - Exemplo: `jpg,jpeg,bmp,ico,gif,png`
- **CDN Static Maximum TTL** *: o máximo TTL (time-to-live), em segundos, para o conteúdo em cache ser entregue.
    - Aceita valores entre `0` e `31536000`. Valor padrão: `31536000`

> - **Browser Image Maximum TTL** *: o máximo TTL (time-to-live), em segundos, para conteúdo armazenado em cache no navegador.

Reviewer: ```suggestion
- **Browser Image Maximum TTL** *: o TTL (time-to-live) máximo, em segundos, para conteúdo armazenado em cache no navegador.
```

---

## Implante o template

Durante a implantação, você poderá acompanhar o processo através de uma janela mostrando os logs. Quando estiver concluido, aparecerá uma nova página confirmando que a edge application foi criada com sucesso.

Esta página mostra as seguintes seções:

- A seção **Access your Edge Application**, que inclui o link para visitar e explorar sua aplicação.
- O **Build Summary** contém o nome e a função da aplicação.
- A aba **Deployment details** pode ser aberta para acessar os logs relacionados à implantação.
- **What do you want to do next?** fornece recomendações sobre opções avançadas para aprimorar sua edge application: **Customize Domain**, **Manage Edge Application**, **View Edge Application Metrics**, **View Edge Application Logs** e o botão **Back to Home**.

:::note
O link para sua edge application permite que você veja como ela fica no navegador. No entanto, leva um certo tempo para propagar e configurá-la nas edge locations da Azion. Pode ser necessário aguardar alguns minutos para que a URL seja ativada e para que a página da aplicação seja efetivamente exibida no navegador.
:::

> ### Principais configurações

Reviewer 2: ```suggestion
### Configurações principais
```

2. No canto superior esquerdo, selecione as três linhas horizontais para abrir o **Products menu**.
3. Na seção de **BUILD**, selecione **Edge Application**.
4. Você será redirecionado para a página de **Edge Application**.
- Ela lista todas as edge applications que você criou.
5. Encontre a aplicação relacionada ao seu template e selecione-a.
6. Na aba **Main Settings**, encontre a seção **Edge Application Modules** e ative o switch dos módulos.

:::caution[Atenção]
Se estes módulos estiverem ativados, a execução deste template pode gerar custos relacionados ao uso. Consulte a [página de preços](https://www.azion.com/pt-br/precos/) para obter mais informações.
:::

---

## Gerencie o template

Considerando que essa configuração inicial pode não ser ideal para sua aplicação, todas as configurações podem ser personalizadas sempre que você precisar usando o RTM.

Para gerenciar e editar as configurações da sua aplicação, siga estas etapas:

1. [Acesse o RTM](https://manager.azion.com/).
2. No canto superior esquerdo, selecione as três linhas horizontais para abrir o **Products menu**.
3. Na seção de **BUILD**, selecione **Edge Application**.

> 4. Você será redirecionado para a página de **Edge Application**.
> - Ela lista todas as edge applications que você criou.
> 5. Encontre a aplicação relacionada ao **Fauna DB Boilerplate* e selecione-a.
> - A lista é organizada em ordem alfabética. Você também pode usar a **barra de busca** localizada no canto superior esquerdo da lista; atualmente, ela é filtrada apenas pelo **Application Name**, ou nome da edge application.

Reviewer: ```suggestion
- Você será redirecionado para a página de **Edge Application**. Ela lista todas as edge applications que você criou.
4. Encontre a aplicação relacionada ao **Image Suite** e selecione-a.
- A lista é organizada em ordem alfabética. Você também pode usar a **barra de busca** localizada no canto superior esquerdo da lista; atualmente, ela é filtrada apenas pelo **Application Name**, ou nome da edge application.
```
